packages/cli/src/commands/test/safe.ts

```diff
@@ -0,0 +1,74 @@
+import { Flags } from '@oclif/core'
+import { MetaTransactionData } from '@safe-global/types-kit'
+import { BaseCommand } from '../../base'
+import { CustomFlags } from '../../utils/command'
+import { performSafeTransaction } from '../../utils/safe'
+
+export default class Safe extends BaseCommand {
```

Comment:
im leaning towards not having this in the release. since it really is just for our by hand test that safe integration worked. 

I think once we convert some more commands that integrate with safe we will have the same thing with those as this gives us.